smsdk/smsdk_entities/machine/machine_v1.py

```diff
@@ -64,3 +65,39 @@ def get_type_from_machine_name(self, machine_source, *args, **kwargs):
             ):
                 machine_type = record["type"]
         return machine_type
+
+    def modify_query_params(self, url, kwargs):
```

Comment:
I would like to see a test added for this

Reply:
Added a unit test to check query params with machine v1 endpoint, although it was already included with all the tests in which `get_machines()` method is calling.